sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -107,13 +107,16 @@ export type CertificateContentType = "application/pem" | "application/x-pkcs12"
 
 // @public
 export interface CertificateIssuer {
+    accountId?: string;
     administratorContacts?: AdministratorContact[];
     createdOn?: Date;
     credentials?: IssuerCredentials;
     enabled?: boolean;
     id?: string;
+    issuerProperties?: IssuerProperties;
     name?: string;
     organizationId?: string;
+    password?: string;
     provider?: string;
```

Comment:
Should be only on `IssuerProperties`.

Reply:
.Net has it at the CertificateIssuer level: https://apiview.azurewebsites.net/Assemblies/Review/0191c2c3616d49b4b0d567adc8668b5a#Azure.Security.KeyVault.Certificates.CertificateIssuer

Reply:
Ahh the provider! Gotcha. My bad

Reply:
Should be good now :)